### ISSUE TITLE
Version 2.4.0(e)

### ISSUE THREAD

**Issue by kmeintsma** [geoffhumphrey/brewcompetitiononlineentry#1343]
I have a more detailed question and can't find anything in the software or in the help system to answer it.

Question 1:
I have a large category (multiple actually) that I want to split into two categories by sweetness for both judging and awards. How can I do this?

It's simple enough to create flights for a table that would be separated by sweetness if I look at the entry information although it's extra work. But awards are reported by table correct? I can't create extra tables as far as I can tell and then assign flights to them based on the characteristics I'm targeting.

Question 2:
I've added some additional judging sessions. When I try to assign judges to a table by session, the software will NOT allow me to select the judges I need. I presume it's because they didn't sign up for the session due to it being created after signups were done. Is there a way around this? I really don't want to go into each judge profile and force select them into a new judging session.

**Comment by geoffhumphrey** [comment]
Hi @kmeintsma - hopefully the following will answer your questions.

**Question 1**
You can spilt into flights like you described. And yes, awards are by table. If you want to have awards for the differing levels of sweetness within a sub-category, my advice would be to:
1. Create a custom "style" for each level of sweetness.
2. Reassign the affected entries to the appropriate custom style.
3. Create a judging table for each level of sweetness.

This way, you'll have medal categories for each.

**Question 2**
Correct. There are a couple of approaches around this.
1. Send a mass email to all judges informing of the newly created judging session(s) and requesting that they update their availability via the Edit My Account function.
2. Just like you mentioned - edit the accounts of those judges whom you know are available yourself as the Admin. 

Not allowing judges to be assigned to sessions where they're explicitly unavailable is a failsafe in the system.

Cheers!

**Comment by kmeintsma** [comment]
Incidentally, I just tested moving entries from one style to another just
to make sure you had not made changes since the last time I tried this.

When the style is changed to the new style, ALL of the data is wiped out.

There is simply no way I can manually type in all of the information for
each entry in order to move them.

Thanks,
Kevin

**Comment by geoffhumphrey** [comment]
This is true and by design, but you do have a good point - Admins should be able to change the style of a given entry if they feel it has been classified incorrectly **without** having to copy/paste required information that was provided by the entrant. 

Disabling the data wipe upon change of style for Admins should be pretty easy to do. Stay tuned.
